docs/developers/environment.md

```diff
@@ -15,7 +15,7 @@ dependencies needed to run a node.
 
 First, make sure to update and upgrade the OS:
 
-```sh
+```bash
```

Comment:
shell and bash are the same thing. Bash is just more added features for shell so I wouldn't modify it for now.